tests/HUBViewControllerTests.m

```diff
@@ -270,6 +270,32 @@ - (void)testDelegateNotifiedOfUpdatedViewModel
     XCTAssertEqualObjects(self.viewModelFromDelegateMethod.navigationItem.title, viewModelNavBarTitleB);
 }
 
+- (void)testReloadViewModel
+{
+    NSString * const viewModelNavBarTitleA = @"View model A";
+    NSString * const viewModelNavBarTitleB = @"View model B";
+    
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> builder) {
+        builder.navigationBarTitle = viewModelNavBarTitleA;
+        return YES;
+    };
+    
+    [self simulateViewControllerLayoutCycle];
+    
+    XCTAssertEqualObjects(self.viewModelFromDelegateMethod.navigationItem.title, viewModelNavBarTitleA);
+    
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> builder) {
+        builder.navigationBarTitle = viewModelNavBarTitleB;
+        return YES;
+    };
+    
+    self.contentReloadPolicy.shouldReload = YES;
```

Comment:
@zhubofei Basically this is what's preventing the reload:
```
if (self.contentReloadPolicy != nil) {
        if (self.previouslyLoadedViewModel != nil) {
            id<HUBViewModel> const previouslyLoadedViewModel = self.previouslyLoadedViewModel;
            
            if (![self.contentReloadPolicy shouldReloadContentForViewURI:self.viewURI currentViewModel:previouslyLoadedViewModel]) {
                return;
            }
        }
    }
```

In `HUBViewModelLoaderImplementation.loadViewModel`.

We can fix it by adding a new method: `loadViewModelRegardlessOfReloadPolicy` or something 😄